src/scripts/components/MultiAvatar.jsx

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class MultiAvatar extends React.Component {
+  renderInitials(initials) {
+    return (
+      <text x="50" y="50" textAnchor="middle" dy="0.35em" fontSize="30">{initials}</text>
+    );
+  }
+
+  formatInitials(name) {
+    const splitName = name.split(' ');
```

Comment:
You might want to put some protections here if the name isn't in the format you expect.

Reply:
Unsure what protections to actually place here, now that I think about it. We're doing the same for the `Avatar` component, and it should work with initials and also full names. Open to suggestions and I'll make the change in both components.